### PR TITLE
Add NES, SNES, GBA, and N64 Resolution packs

### PR DESCRIPTION
**Pull request by Zalnor**
Have fun looking through this, or trying to validate the title ids.

### CODE REVIEW

File: Resolutions/VC_GBA_resolution/rules.txt [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+name = "Game Boy Advanced VC resolution"
```
Comment (slashiee) [inline Resolutions/VC_GBA_resolution/rules.txt]: Resolution

File: Resolutions/VC_GBA_resolution/rules.txt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+name = "Game Boy Advanced VC resolution"
+path = VC/GBA/Resolution
```
Comment (slashiee) [inline Resolutions/VC_GBA_resolution/rules.txt]: "Virtual Console/GBA/Resolution"

File: Resolutions/VC_GBA_resolution/rules.txt [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+name = "Game Boy Advanced VC resolution"
+path = VC/GBA/Resolution 
+version = 3
```
Comment (slashiee) [inline Resolutions/VC_GBA_resolution/rules.txt]: copy & paste description from other res packs

File: Resolutions/VC_N64_resolution/rules.txt [hunk 4]
```diff
@@ -0,0 +1,175 @@
+[Definition]
+titleIds = 50000101FEA00,50000101FE900,5000010199200,5000010199100,5000010199300,5000010199500,5000010199400,5000010199700,5000010199600,5000010199800,5000010199B00,5000010199A00,5000010199C00,50000101A0E00,50000101A0D00,50000101A1000,50000101A5900,50000101A5800,50000101A5B00,50000101A5A00,50000101AC400,50000101AC600,50000101AC500,50000101B4600,50000101B4500,50000101B4800,50000101B4700,50000101BAF00,50000101BAE00,50000101BB000,50000101BB200,50000101BB100,50000101C5D00,50000101C5C00,50000101C5E00,50000101DA800,50000101DA700,50000101DA900,50000101E6500,50000101E6400,50000101E6700,50000101E6600,50000101EBB00,50000101FE800,50000101FE700
+name = "Nintendo 64 VC resolution"
```
Comment (slashiee) [inline Resolutions/VC_N64_resolution/rules.txt]: Resolution

File: Resolutions/VC_N64_resolution/rules.txt [hunk 5]
```diff
@@ -0,0 +1,175 @@
+[Definition]
+titleIds = 50000101FEA00,50000101FE900,5000010199200,5000010199100,5000010199300,5000010199500,5000010199400,5000010199700,5000010199600,5000010199800,5000010199B00,5000010199A00,5000010199C00,50000101A0E00,50000101A0D00,50000101A1000,50000101A5900,50000101A5800,50000101A5B00,50000101A5A00,50000101AC400,50000101AC600,50000101AC500,50000101B4600,50000101B4500,50000101B4800,50000101B4700,50000101BAF00,50000101BAE00,50000101BB000,50000101BB200,50000101BB100,50000101C5D00,50000101C5C00,50000101C5E00,50000101DA800,50000101DA700,50000101DA900,50000101E6500,50000101E6400,50000101E6700,50000101E6600,50000101EBB00,50000101FE800,50000101FE700
+name = "Nintendo 64 VC resolution"
+path = VC/N64/Resolution
```
Comment (slashiee) [inline Resolutions/VC_N64_resolution/rules.txt]: "Virtual Console/N64/Resolution"

File: Resolutions/VC_NES_resolution/rules.txt [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+name = "Nintendo Entertainment System VC Resolution"
```
Comment (slashiee) [inline Resolutions/VC_NES_resolution/rules.txt]: Resolution

File: Resolutions/VC_NES_resolution/rules.txt [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+name = "Nintendo Entertainment System VC Resolution"
+path = VC/NES/Resolution
```
Comment (slashiee) [inline Resolutions/VC_NES_resolution/rules.txt]: "Virtual Console/NES/Resolution"

File: Resolutions/VC_SNES_resolution/rules.txt [hunk 8]
```diff
@@ -0,0 +1,144 @@
+[Definition]
+titleIds = 500001012A800,500001012E900,500001012EC00,500001012EE00,500001012ED00,500001012EF00,5000010130200,5000010130400,5000010130300,5000010130500,5000010130700,5000010130600,5000010130800,5000010130A00,5000010130900,5000010130B00,5000010108F00,5000010108E00,5000010109000,5000010109200,5000010109100,5000010109300,5000010109500,5000010109400,5000010109600,5000010109800,5000010109700,5000010109900,5000010109D00,5000010109C00,5000010109E00,5000010109F00,500001010A100,500001010A000,500001010A200,500001010A400,500001010A600,500001010A500,500001010A700,500001010A900,500001010A800,500001010AA00,5000010114C00,5000010114B00,5000010114D00,5000010115000,5000010114F00,5000010115100,5000010115300,5000010115200,5000010115400,5000010115A00,5000010115B00,5000010119A00,5000010119900,5000010119B00,5000010119D00,5000010119C00,5000010119E00,500001011A000,5000010119F00,500001011A100,500001011A200,500001011A400,500001011A300,500001011A500,500001012A200,500001012A300,500001012A600,500001012A500,500001012AD00,500001012A700,500001014B300	
+name = "Super Nintendo VC Resolution"
```
Comment (slashiee) [inline Resolutions/VC_SNES_resolution/rules.txt]: Resolution

File: Resolutions/VC_SNES_resolution/rules.txt [hunk 9]
```diff
@@ -0,0 +1,144 @@
+[Definition]
+titleIds = 500001012A800,500001012E900,500001012EC00,500001012EE00,500001012ED00,500001012EF00,5000010130200,5000010130400,5000010130300,5000010130500,5000010130700,5000010130600,5000010130800,5000010130A00,5000010130900,5000010130B00,5000010108F00,5000010108E00,5000010109000,5000010109200,5000010109100,5000010109300,5000010109500,5000010109400,5000010109600,5000010109800,5000010109700,5000010109900,5000010109D00,5000010109C00,5000010109E00,5000010109F00,500001010A100,500001010A000,500001010A200,500001010A400,500001010A600,500001010A500,500001010A700,500001010A900,500001010A800,500001010AA00,5000010114C00,5000010114B00,5000010114D00,5000010115000,5000010114F00,5000010115100,5000010115300,5000010115200,5000010115400,5000010115A00,5000010115B00,5000010119A00,5000010119900,5000010119B00,5000010119D00,5000010119C00,5000010119E00,500001011A000,5000010119F00,500001011A100,500001011A200,500001011A400,500001011A300,500001011A500,500001012A200,500001012A300,500001012A600,500001012A500,500001012AD00,500001012A700,500001014B300	
+name = "Super Nintendo VC Resolution"
+path = VC/SNES/Resolution
```
Comment (slashiee) [inline Resolutions/VC_SNES_resolution/rules.txt]: "Virtual Console/SNES/Resolution"

File: Resolutions/VC_N64_resolution/rules.txt [hunk 10]
```diff
@@ -0,0 +1,175 @@
+[Definition]
+titleIds = 50000101FEA00,50000101FE900,5000010199200,5000010199100,5000010199300,5000010199500,5000010199400,5000010199700,5000010199600,5000010199800,5000010199B00,5000010199A00,5000010199C00,50000101A0E00,50000101A0D00,50000101A1000,50000101A5900,50000101A5800,50000101A5B00,50000101A5A00,50000101AC400,50000101AC600,50000101AC500,50000101B4600,50000101B4500,50000101B4800,50000101B4700,50000101BAF00,50000101BAE00,50000101BB000,50000101BB200,50000101BB100,50000101C5D00,50000101C5C00,50000101C5E00,50000101DA800,50000101DA700,50000101DA900,50000101E6500,50000101E6400,50000101E6700,50000101E6600,50000101EBB00,50000101FE800,50000101FE700
+name = "Nintendo 64 VC resolution"
+path = VC/N64/Resolution 
+version = 3
```
Comment (slashiee) [inline Resolutions/VC_N64_resolution/rules.txt]: copy & paste description from other res packs

File: Resolutions/VC_NES_resolution/rules.txt [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+name = "Nintendo Entertainment System VC Resolution"
+path = VC/NES/Resolution 
+version = 3
```
Comment (slashiee) [inline Resolutions/VC_NES_resolution/rules.txt]: copy & paste description from other res packs

File: Resolutions/VC_SNES_resolution/rules.txt [hunk 12]
```diff
@@ -0,0 +1,144 @@
+[Definition]
+titleIds = 500001012A800,500001012E900,500001012EC00,500001012EE00,500001012ED00,500001012EF00,5000010130200,5000010130400,5000010130300,5000010130500,5000010130700,5000010130600,5000010130800,5000010130A00,5000010130900,5000010130B00,5000010108F00,5000010108E00,5000010109000,5000010109200,5000010109100,5000010109300,5000010109500,5000010109400,5000010109600,5000010109800,5000010109700,5000010109900,5000010109D00,5000010109C00,5000010109E00,5000010109F00,500001010A100,500001010A000,500001010A200,500001010A400,500001010A600,500001010A500,500001010A700,500001010A900,500001010A800,500001010AA00,5000010114C00,5000010114B00,5000010114D00,5000010115000,5000010114F00,5000010115100,5000010115300,5000010115200,5000010115400,5000010115A00,5000010115B00,5000010119A00,5000010119900,5000010119B00,5000010119D00,5000010119C00,5000010119E00,500001011A000,5000010119F00,500001011A100,500001011A200,500001011A400,500001011A300,500001011A500,500001012A200,500001012A300,500001012A600,500001012A500,500001012AD00,500001012A700,500001014B300	
+name = "Super Nintendo VC Resolution"
+path = VC/SNES/Resolution 
+version = 3
```
Comment (slashiee) [inline Resolutions/VC_SNES_resolution/rules.txt]: copy & paste description from other res packs

**Comment by slashiee** [comment]
Folder names should also match the other Resolution Packs:
```
VCGBA_Resolution
VCN64_Resolution
```
etc etc.

**Comment by Zalnor** [comment]
> 
> 
> Folder names should also match the other Resolution Packs:
> 
> ```
> VCGBA_Resolution
> VCN64_Resolution
> ```
> 
> etc etc.

I have no clue how to change this in github :/

**Comment by slashiee** [comment]
Hm, alright. If I remember to I can rename after it's merged. Just needs testing now

**Pull request merged**